src/main/resources/db/changelog/20190116_1301_update_facilities_communities_reports.mysql.sql

```diff
@@ -0,0 +1,52 @@
707371007e00550000ea627371007e005a00000000ff00000070707070707371007e004d3f00000071007e047f71007e047f707371007e00690000ea627371007e005a00000000ff00000070707070707371007e004d3f00000071007e047f71007e047f707371007e006d0000ea627371007e005a00000000ff00000070707070707371007e004d3f00000071007e047f71007e047f7371007e00560000ea627070707071007e047d7070707071007e043a74000a5461626c6520315f43487371007e00717070707071007e047d7070707070707070707070707070707070707070707371007e041f0000ea62007371007e005a00000000ffffffff7070707071007e001170707070707070707070707070707070707371007e0050707371007e00540000ea627371007e005a00000000ff00000070707070707371007e004d3f00000071007e049471007e049471007e0492707371007e00640000ea627371007e005a00000000ff00000070707070707371007e004d3f00000071007e049471007e0494707371007e00550000ea627371007e005a00000000ff00000070707070707371007e004d3f00000071007e049471007e0494707371007e00690000ea627371007e005a00000000ff00000070707070707371007e004d3f00000071007e049471007e0494707371007e006d0000ea627371007e005a00000000ff00000070707070707371007e004d3f00000071007e049471007e04947371007e00560000ea627070707071007e04927070707071007e043a74000a5461626c6520315f54447371007e00717070707071007e049270707070707070707070707070707070707070707070707371007e00127371007e001b0000000e77040000000e7371007e00260000ea62000000320001000000000000015e00000000000000007071007e001171007e04a770707070707071007e003e70707371007e0040707371007e001b00000001770400000001740021636f6d2e6a6173706572736f66742e73747564696f2e756e69742e686569676874787371007e00453f400000000000037708000000040000000171007e04ac740005706978656c787071007e00497371007e004b9e48704660ec59ecea3b8fe8c2884c620000ea6270707371007e004d418000007070707070707070707371007e0050707371007e00540000ea627070707071007e04b171007e04b171007e04a9707371007e00640000ea627070707071007e04b171007e04b1707371007e00550000ea627070707071007e04b171007e04b1707371007e00690000ea627070707071007e04b171007e04b1707371007e006d0000ea627070707071007e04b171007e04b17070707371007e00717070707071007e04a9707070707070707070707071007e0075740011466163696c6974696573205265706f72747371007e00260000ea62000000000001000000000000000000000000000000007071007e001171007e04a770707070707071007e003e70707371007e0040707371007e001b000000027704000000027400256e65742e73662e6a61737065727265706f7274732e6578706f72742e6a736f6e2e706174687400256e65742e73662e6a61737065727265706f7274732e6578706f72742e6a736f6e2e64617461787371007e00453f400000000000037708000000040000000271007e04bc74001a636f6c4d6f64656c2e666163696c69747949642e48656164657271007e04bd74000b466163696c697479204944787071007e00497371007e004baff4c7a548ff7988d7f916b0b84640700000ea627070707070707070707070707371007e0050707371007e00540000ea627070707071007e04c271007e04c271007e04b9707371007e00640000ea627070707071007e04c271007e04c2707371007e00550000ea627070707071007e04c271007e04c2707371007e00690000ea627070707071007e04c271007e04c2707371007e006d0000ea627070707071007e04c271007e04c27070707371007e00717070707071007e04b9707070707070707070707070707371007e00260000ea6200000000
```

Comment:
remove the second semicolon and add newline at the end of the file

Reply:
fixed